packages/toolkit/src/query/tests/buildHooks.test.tsx

```diff
@@ -1669,7 +1670,8 @@ describe('hooks tests', () => {
         expect(screen.getByTestId('isFetching').textContent).toBe('false'),
       )
 
-      userEvent.hover(screen.getByTestId('highPriority'))
+      await userEvent.hover(screen.getByTestId('highPriority'))
```

Comment:
Since [Release v14.0.0 · testing-library/user-event](https://github.com/testing-library/user-event/releases/tag/v14.0.0) `userEvent.hover` returns a promise, so we need to await here.